packages/timetable-planner/src/index.ts

```diff
@@ -0,0 +1,120 @@
+import { courseCombinations, courseMeetingSectionCombinations, } from "./combinations/combinations"
+
+const checkOverlap = (timetable: Timetable): boolean => {
```

Comment:
Merge the changes from ``planner-generate-timetable`` into this branch so this file is up to date.